### PR TITLE
[TRAFWKPR-18] Add mapcontrols

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/MapControls/MapControls.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import swissbounds from '../../img/swissbounds.png';
+import './MapControls.scss';
+
+const propTypes = {
+  map: PropTypes.instanceOf(OLMap).isRequired,
+  t: PropTypes.func.isRequired,
+};
+
+const MapControls = ({ map, t }) => {
+  return (
```
Comment (mhaertwig) [inline src/components/MapControls/MapControls.js]: The return is not needed.

**Comment by mhaertwig** [comment]
A netlify preview would be cool ;-)

**Comment by edellucien** [comment]
There is no git hooks yet on this project :cry: 
I will add this to the MR as well.

**Comment by edellucien** [comment]
Ready to be merged @mhaertwig

File: src/components/MapControls/MapControls.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      noCenterAfterDrag
+      colorOrStyleFunc={[0, 61, 133]}
+    />
+    <FitExtent
+      map={map}
+      title={t('Ganze Schweiz')}
+      extent={CONF.swissExtent}
+      className="wkp-fit-extent"
+    >
+      {<img src={swissbounds} alt={t('Ganze Schweiz')} />}
```
Comment (mhaertwig) [inline src/components/MapControls/MapControls.js]: @edellucien are the curly brackets needed?

Comment (edellucien) [reply src/components/MapControls/MapControls.js]: Indeed it is unnecessary, I remove it :+1:

**Pull request merged**